client/app/calendar/tasks/components/AssigneeField.tsx

```diff
@@ -0,0 +1,101 @@
+  const [selectedAssigneeName, setSelectedAssigneeName] = useState<
+    string | undefined
+  >(undefined)
+
+  const [isDropdownOpen, setDropdownOpen] = useState(false)
+
+  return (
+    <FormField
+      control={control}
+      name={name}
+      render={() => (
+        <FormItem className='flex w-full flex-col'>
+          <FormLabel>
+            Assignee <span className='text-destructive'>*</span>
+          </FormLabel>
+          <FormControl>
+            <Controller
+              control={control}
+              name={name}
+              render={({ field }) => (
+                <DropdownMenu
+                  open={isDropdownOpen}
+                  onOpenChange={setDropdownOpen}
+                >
+                  <DropdownMenuTrigger asChild>
+                    <button
+                      className={`flex h-10 w-full items-center justify-between rounded-md border border-input bg-slate-50 px-3 py-2 text-left text-sm ring-offset-background focus:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50 ${!field.value ? 'text-slate-400' : ''} ${
+                        isDropdownOpen
+                          ? 'border border-input ring-2 ring-ring ring-offset-2 ring-offset-background'
+                          : ''
+                      }`}
+                    >
+                      {selectedAssigneeName || 'Select assignee'}
+                      <CaretSortIcon className='h-6 w-6 opacity-50' />
+                    </button>
+                  </DropdownMenuTrigger>
+                  <DropdownMenuContent
+                    style={{
+                      // To set the same width as the trigger (https://www.radix-ui.com/primitives/docs/components/dropdown-menu)
+                      width: 'var(--radix-dropdown-menu-trigger-width)',
+                    }}
+                  >
+                    {houseMembers?.map((member) => (
+                      <DropdownMenuCheckboxItem
+                        key={member.id}
+                        checked={field.value === member.id}
+                        onCheckedChange={() => {
+                          if (field.value !== member.id) {
+                            field.onChange(member.id)
+                            setSelectedAssigneeName(member.name)
+                          } else {
+                            field.onChange(undefined)
+                            setSelectedAssigneeName(undefined)
+                          }
+                        }}
+                        className={`${
+                          field.value === member.id
+                            ? 'w-full bg-primary text-primary-foreground focus:bg-primary focus:text-primary-foreground'
+                            : ''
+                        }`}
+                      >
+                        {member.name}
+                      </DropdownMenuCheckboxItem>
```

Comment:
I feel too much padding-left!
<img width="446" alt="Screenshot 2024-03-18 at 4 05 21 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/ea459885-e01a-44e4-8dbe-4e2453fc53f9">

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/80/commits/4e979c86b5fe1ed9289735e158736da5ddea5fc0